docker-compose-releng.yml

```diff
@@ -0,0 +1,84 @@
+version: "2.1"
+
+# TODO: what links do we need?
+services:
+  releng_db:
+    image: postgres:9.6
```

Comment:
This is significantly more work than using the Postgres image from the Docker library, which automatically handles creating users, databases, etc. How about pinning to a more specific version (9.6.2) as a compromise?